lib/sidekiq/api.rb

```diff
@@ -700,6 +712,7 @@ def self.timeout
   #
   class ProcessSet
     include Enumerable
+    extend RedisIterator
```

Comment:
include here also.  refactor cleanup to be an instance method.  Push back if I'm missing something.